packages/hoppscotch-common/assets/scss/styles.scss

```diff
@@ -137,10 +137,10 @@ a {
 
 .cm-tooltip {
   .tippy-box {
-    @apply shadow-none;
+    @apply shadow-none #{!important};
     @apply fixed;
     @apply inline-flex;
-    @apply -mt-8;
+    @apply -mt-7.5;
```

Comment:
It is better to set the margin to either 7 or 8 since Tailwind doesn't support 7.5. We are moving back to Tailwind in the next point release and this style won't be applied.

Reply:
Neither 7 or 8 suits the desired outcome. Hence 7.5. Tailwind do supports custom values like m-[7.5rem], hence migrating to Tailwind won't be an issue.